AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -133,6 +133,8 @@ foxnews.com#%#//scriptlet('set-constant', 'Taplytics.runningExperiments', 'false
 foxnews.com#%#//scriptlet('adjust-setTimeout', 'Taplytics', '*', '0.001')
 foxnews.com#%#//scriptlet('remove-class', 'gated-overlay', '.article-gating')
 foxnews.com#%#//scriptlet('remove-class', 'has-gated-overlay|gated-article-body', '.gated-article-body')
+! https://github.com/AdguardTeam/AdguardFilters/issues/180279
+it-world.ru###bx_sender_subform_RPnsty
```

Comment:
Please move the rule like `@piquark6046` [said](https://github.com/AdguardTeam/AdguardFilters/pull/180813#pullrequestreview-2098499856). Just the rule without the task comment.
Btw, better block it like this.
```adblock
it-world.ru##.subscribe-detail
```